src/sync/mod.rs

```diff
@@ -827,7 +1545,7 @@ mod tests {
                 exp_new_sync_head: None,
             },
             Case {
-                name: "nop push, pulls new state w/lesser mutation id -> beginsync errors",
+                name: "nop push, pulls new state w/lesser mutation id -> beginpull errors",
```

Comment:
fyi when push and pull tests are separated we should end up with fewer test cases because we don't have to do the product of |{potential push outcomes}|x|{potential pull outcomes}|